vminfo_parser/visualizer.py

```diff
@@ -255,7 +258,22 @@ def _get_colors(os_names: list[str]) -> list[ColorType]:
     chosen_colors: list[ColorType] = []
     for os in os_names:
         if os in supported_os_names:
-            chosen_colors.append(mcolors.to_rgba(const.SUPPORTED_OS_COLORS.get(os)))
+            chosen_colors.append(mcolors.to_rgba(const.SUPPORTED_OS_COLORS[os]))
         else:
             chosen_colors.append(raw_colors.pop())
     return chosen_colors
+
+
+def _color_diff(color_a: ColorType, color_b: ColorType) -> float:
+    """Calculate absolute value of the distance between two colors.
+
+    Args:
+        a (ColorType): color to calculate difference
+        b (ColorType): color to calculate difference
+
+    Returns:
+        float: absolute value of the distance between a and b in rgba color space
+    """
+    color_tuples = (mcolors.to_rgba(color_a), mcolors.to_rgba(color_b))
+    color_values: Iterable[tuple[float, float]] = zip(*color_tuples)
+    return math.fsum([abs(a - b) for a, b in color_values])
```

Comment:
What is the need for this function? I understand what it is doing, but I am not sure why it is needed, when it wasn't needed before